libkbfs/folder_block_ops.go

```diff
@@ -1863,6 +1863,15 @@ func (fbo *folderBlockOps) startSyncWrite(ctx context.Context,
 		dirtyDe = &de
 	}
 
+	// Leave a copy of the syncOp in `unrefCache`, since it may be
```

Comment:
Ah, I see. Yeah, in that case, I think leaving it as-is is fine, although a comment about it above `AddOp` might be warranted.

However, updating an existing op in place makes me nervous. Mind updating into a fresh variable and then assigning?